staging/src/k8s.io/client-go/examples/README.md

```diff
@@ -19,10 +19,13 @@ for client-go.
 
 - [**Work queues**](./workqueue): Create a hotloop-free controller with the
   rate-limited workqueue and the [informer framework][informer].
-- [**Third-party resources (deprecated)**](./third-party-resources-deprecated):
+- [**Custom Resources**](https://git.k8s.io/apiextensions-apiserver/examples/client-go):
   Register a custom resource type with the API, create/update/query this custom
-  type, and write a controller drives the cluster state based on the changes to
+  type, and write a controller that drives the cluster state based on the changes to
   the custom resources.
+- [**Third-party resources (deprecated)**](./third-party-resources-deprecated):
+  Register a third-party resource type with the API, create/update/query this third-party
```

Comment:
Maybe explicitly point out that CRD is the successor of TPR, and switch the order of TRP entry and CRD entry.